src/Avalonia.Visuals/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,12 @@
+Compat issues with assembly Avalonia.Visuals:
+EnumValuesMustMatch : Enum value 'Avalonia.Media.TextWrapping Avalonia.Media.TextWrapping.NoWrap' is (System.Int32)0 in the implementation but (System.Int32)1 in the contract.
+EnumValuesMustMatch : Enum value 'Avalonia.Media.TextWrapping Avalonia.Media.TextWrapping.Wrap' is (System.Int32)1 in the implementation but (System.Int32)2 in the contract.
+EnumValuesMustMatch : Enum value 'Avalonia.Media.TextWrapping Avalonia.Media.TextWrapping.WrapWithOverflow' is (System.Int32)2 in the implementation but (System.Int32)0 in the contract.
+MembersMustExist : Member 'public void Avalonia.Media.TextFormatting.GenericTextParagraphProperties..ctor(Avalonia.Media.TextFormatting.TextRunProperties, Avalonia.Media.TextAlignment, Avalonia.Media.TextWrapping, Avalonia.Media.TextTrimming, System.Double)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public Avalonia.Media.TextTrimming Avalonia.Media.TextFormatting.GenericTextParagraphProperties.TextTrimming.get()' does not exist in the implementation but it does exist in the contract.
+CannotAddAbstractMembers : Member 'public System.Boolean Avalonia.Media.TextFormatting.TextLine.HasCollapsed' is abstract in the implementation but is missing in the contract.
+CannotAddAbstractMembers : Member 'public Avalonia.Media.TextFormatting.TextLine Avalonia.Media.TextFormatting.TextLine.Collapse(Avalonia.Media.TextFormatting.TextCollapsingProperties[])' is abstract in the implementation but is missing in the contract.
+CannotAddAbstractMembers : Member 'public System.Boolean Avalonia.Media.TextFormatting.TextLine.HasCollapsed.get()' is abstract in the implementation but is missing in the contract.
+MembersMustExist : Member 'public void Avalonia.Media.TextFormatting.TextLineMetrics..ctor(Avalonia.Size, System.Double, Avalonia.Media.TextFormatting.TextRange)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public Avalonia.Media.TextTrimming Avalonia.Media.TextFormatting.TextParagraphProperties.TextTrimming.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
This was newly introduced after the `0.9` release so this is wrong.

Reply:
@Gillibald this is saying that these apis have changed since 0.10.0-preview1 and we are accepting those changes.